test/test_model_configuration.py

```diff
@@ -8,7 +8,7 @@
 @pytest.fixture(scope="function", autouse=True)
 def init_config(request):
     configuration = Configuration('TestConfig.db')
-    db_file_path = os.environ['USER_HIDDEN'] + 'TestConfig.db'
+    db_file_path = os.environ['USER_ONKODICOM_HIDDEN'] + 'TestConfig.db'
```

Comment:
I will wait for all the unit tests to be merged into master before making anymore changes because many unit tests make use of the environment variable.